charts/shoot-cloud-config/charts/original/templates/kubelet/_kubelet-config-file

```diff
@@ -14,6 +14,9 @@ authorization:
   webhook:
     cacheAuthorizedTTL: 5m0s
     cacheUnauthorizedTTL: 30s
+{{- if semverCompare "< 1.12" .kubernetes.version }}
+cAdvisorPort: 0
```

Comment:
It seems that they haven't ported the `cAdvisorPort` to the `KubeletConfiguration` resource (https://github.com/kubernetes/kubernetes/blob/v1.10.3/pkg/kubelet/apis/kubeletconfig/v1beta1/types.go) as they wanted to remove it anyway in future releases, hence, it's not needed here?

Reply:
fixed in 01916d9c35129f1ddae0b602c9dab7293c5d9146